.gitignore

```diff
@@ -35,6 +32,7 @@ docs/_build/
 
 # log dirs
 **/logs
+cmd/secrets-config/res/
```

Comment:
Why add this? It shouldn't exist and if some change is made where we do want it, this will cause some issues.

Reply:
it exists on my local box for some reason and so I want to ignore it.   As of today, we shouldn't check-in anything from that directory accidentally as that utility doesn't have any resource file associated with it.  
With that being said though, I am ok to remove this.

Reply:
Please remove. You shouldn't have one locally.

Reply:
removed.